src/wp-admin/custom-background.php

```diff
@@ -247,33 +251,34 @@ public function admin_page() {
 <tr>
 <th scope="row"><?php _e( 'Preview' ); ?></th>
 <td>
-	<?php
-	if ( $this->admin_image_div_callback ) {
-		call_user_func( $this->admin_image_div_callback );
-	} else {
-		$background_styles = '';
-		if ( $bgcolor = get_background_color() )
-			$background_styles .= 'background-color: #' . $bgcolor . ';';
-
-		$background_image_thumb = get_background_image();
-		if ( $background_image_thumb ) {
-			$background_image_thumb = esc_url( set_url_scheme( get_theme_mod( 'background_image_thumb', str_replace( '%', '%%', $background_image_thumb ) ) ) );
-			$background_position_x = get_theme_mod( 'background_position_x', get_theme_support( 'custom-background', 'default-position-x' ) );
-			$background_position_y = get_theme_mod( 'background_position_y', get_theme_support( 'custom-background', 'default-position-y' ) );
-			$background_size = get_theme_mod( 'background_size', get_theme_support( 'custom-background', 'default-size' ) );
-			$background_repeat = get_theme_mod( 'background_repeat', get_theme_support( 'custom-background', 'default-repeat' ) );
-			$background_attachment = get_theme_mod( 'background_attachment', get_theme_support( 'custom-background', 'default-attachment' ) );
-
-			// Background-image URL must be single quote, see below.
-			$background_styles .= " background-image: url('$background_image_thumb');"
+		<?php
+		if ( $this->admin_image_div_callback ) {
+			call_user_func( $this->admin_image_div_callback );
+		} else {
+			$background_styles = '';
+			if ( $bgcolor = get_background_color() ) {
```

Comment:
Standards say not to assign value in an if statement.

Reply:
Agreed and I expect this will get picked up in a future rule addition.

Reply:
As per review meeting, to be addressed in future PR.